### PR TITLE
Feature/revamp logging

### PR DESCRIPTION
**Pull request by pacanada**
Revamping logging module:
- Allow the user to disable the rich handler (it could cause problems in docker or jenkins stdout)
- Allow user to specify custom logging format

### CODE REVIEW

File: dfds_ds_toolbox/logging/logging.py [hunk 1]
```diff
@@ -49,28 +52,45 @@ def init_logger(
...
+        If you want to use a custom log format use:
+        >>> logger = init_logger(stream_level="INFO", log_format="%(module)s - %(message)s")
+        >>> logger.info("Custom log format.")
+
     """
 
     # Set up logger
     logger = logging.getLogger(name)
     logger.setLevel(logging.DEBUG)
+    if rich_handler_enabled and log_format is not None:
```
Comment (KPLauritzen) [inline dfds_ds_toolbox/logging/logging.py:69]: Maybe 
```python
if rich_handler_enabled and log_format: 
```
instead?

**Review by nybyhansen — APPROVED** [review]
Nothing more to add - I agree with the comments from Kasper.

File: tests/test_logging.py [hunk 2]
```diff
@@ -24,6 +24,20 @@ def debug_logger():
     return init_logger(stream_level="DEBUG")
 
 
+@pytest.fixture
+def custom_format_logger():
```
[the remark this answers is not in the document]
Comment (pacanada) [reply tests/test_logging.py]: Done, the changes do not appears on the PR directly as  in Azure. Dont know if I have to wait some time until the PR is updated

**Review by KPLauritzen — APPROVED** [review]
LGTM

**Comment by kaqmak** [comment]
In general I don't like that the formatter and handler is something stated inside the code.  Instead, I would suggest writing a `logging.yaml` that your logger then loads (see https://docs.python.org/3/howto/logging.html#configuring-logging)

**Pull request closed without merging**